package.json

```diff
@@ -56,9 +56,9 @@
     "jest": "^29.7.0",
     "lodash": "^4.17.21",
     "microtime": "^3.1.1",
-    "prettier": "^2.8.8",
-    "ts-node": "^10.9.1",
-    "typescript": "^5.2.2"
+    "prettier": "^3.2.5",
```

Comment:
Did our prettier config work ok with prettier 3?

Reply:
Yep! The latest prettier did introduce one new rule which I'm not a fan of: commas after the last argument in a function call. That's the bulk of changes in this PR :(